Browser-storage.js

```diff
@@ -1,7 +1,7 @@
 const contactForm = document.querySelector('#signup');
-const content = JSON.parse(window.localStorage.getItem('storage'));
+const content = JSON.parse(window.localStorage.getItem('datastorage'));
 function check() {
-  if (localStorage.getItem('storage')) {
+  if (localStorage.getItem('datastorage')) {
     contactForm.elements.fullName.value = content.fullName;
     contactForm.elements.email.value = content.email;
     contactForm.elements.message.value = content.message;
```

Comment:
- [ ] Good job on the functionality, But there is a little issue. The name and email field is being saved to local storage and getting back to the from, But the message field is not working expected. Please make sure your local storage is working for messages as well.

### See the images for reference:

![image](https://user-images.githubusercontent.com/83803414/157841489-749ce7f9-5841-4c3b-8154-b6c7cad960dd.png)

![image](https://user-images.githubusercontent.com/83803414/157841549-ab222d1a-dc40-4533-80a8-7e5d2660543d.png)

Reply:
Ok. Can you check again. Maybe you did not write anything in `textarea` box.

Reply:
![datastorage for message in textarea box](https://user-images.githubusercontent.com/86780082/157853296-62969910-c795-47e7-a692-f342d1e4b42d.PNG)

Reply:
> Ok. Can you check again. Maybe you did not write anything in `textarea` box.

@Mikelobi I tried writing content to every field but for the message, the local storage functionality was not working. If you are sure it's working now, You can go for another review. I hope it helps 😄